src/basic/fileio.h

```diff
@@ -15,17 +15,18 @@
 #define LONG_LINE_MAX (1U*1024U*1024U)
 
 typedef enum {
-        WRITE_STRING_FILE_CREATE                = 1 << 0,
-        WRITE_STRING_FILE_TRUNCATE              = 1 << 1,
-        WRITE_STRING_FILE_ATOMIC                = 1 << 2,
-        WRITE_STRING_FILE_AVOID_NEWLINE         = 1 << 3,
-        WRITE_STRING_FILE_VERIFY_ON_FAILURE     = 1 << 4,
-        WRITE_STRING_FILE_VERIFY_IGNORE_NEWLINE = 1 << 5,
-        WRITE_STRING_FILE_SYNC                  = 1 << 6,
-        WRITE_STRING_FILE_DISABLE_BUFFER        = 1 << 7,
-        WRITE_STRING_FILE_NOFOLLOW              = 1 << 8,
-        WRITE_STRING_FILE_MKDIR_0755            = 1 << 9,
-        WRITE_STRING_FILE_MODE_0600             = 1 << 10,
+        WRITE_STRING_FILE_CREATE                                = 1 << 0,
+        WRITE_STRING_FILE_TRUNCATE                              = 1 << 1,
+        WRITE_STRING_FILE_ATOMIC                                = 1 << 2,
+        WRITE_STRING_FILE_AVOID_NEWLINE                         = 1 << 3,
+        WRITE_STRING_FILE_VERIFY_ON_FAILURE                     = 1 << 4,
+        WRITE_STRING_FILE_VERIFY_IGNORE_NEWLINE                 = 1 << 5,
+        WRITE_STRING_FILE_SYNC                                  = 1 << 6,
+        WRITE_STRING_FILE_DISABLE_BUFFER                        = 1 << 7,
+        WRITE_STRING_FILE_NOFOLLOW                              = 1 << 8,
+        WRITE_STRING_FILE_MKDIR_0755                            = 1 << 9,
+        WRITE_STRING_FILE_MODE_0600                             = 1 << 10,
+        WRITE_STRING_FILE_SUPPRESS_REDUNDANT_VIRTUAL            = 1 << 11,
```

Comment:
please add a single space after the VIRTUAL only and then align the rest of the enum table to that. now you added a lot of whitespace for no reason